code/object/collideshipship.cpp

```diff
@@ -473,6 +473,33 @@ static bool check_subsystem_landing_allowed(ship_info *heavy_sip, collision_info
 	return false;
 }
 
+/**
+ * Helper function that calculates the next time this ship should be allowed to collide with a client player on multiplayer.
+ */
+int calculate_next_multiplayer_client_collision_time(float impulse_magnitude)
```

Comment:
The function should be declared static, since it's not used outside of this file.

Reply:
Should be fixed.